src/axom/quest/InOutOctree.hpp

```diff
@@ -17,8 +17,7 @@
 #include "axom/slam.hpp"
 #include "axom/primal.hpp"
 #include "axom/mint.hpp"
-
-#include "axom/spin/SpatialOctree.hpp"
+#include "axom/spin.hpp"
```

Comment:
I might be remembering our conversation wrongly but I thought we decided not to use the unified headers in our code due to compilation time concerns.  Am I or have we decided against that decision?

Reply:
My recollection was that within a component, we should try to use the unified headers for the dependent components, but not for itself.

(and perhaps we should revisit this if compilation time increases, but it certainly simplifies header inclusions until then...  It is also possible that using unified headers can accelerate compilation times if the compiler is always including things in the same order.).